lib/commons/dom/create-grid.js

```diff
@@ -254,65 +254,61 @@ function isFlexOrGridContainer(vNode) {
  * zIndex values for each stacking context parent.
  * @param {VirtualNode} vNode
  * @param {VirtualNode} parentVNode
- * @param {Number} nodeIndex
+ * @param {Number} index
```

Comment:
Can you give this a better name? If this is no longer an index of the node in the flat tree, what is it? I don't think it's an index at all anymore. You can't use this number to look something up in a table / tree.